src/Symfony/Component/Scheduler/Generator/CheckpointInterface.php

```diff
@@ -18,6 +18,8 @@ interface CheckpointInterface
 {
     public function acquire(\DateTimeImmutable $now): bool;
 
+    public function from(): \DateTimeImmutable;
```

Comment:
Not a BC break as the component is experimental in 6.3, but this should be described in the component CHANGELOG file.

Reply:
Well, technically it is a BC break, but an allowed one. the changelog needs to explicitly mark it as a BC break.

Reply:
If we didn't want this BC break, we could store 2 checkpoints, one for the start state (I guess index would always be -1) and other for latest state (=the current). Let me know if you'd prefer that

Reply:
As we will merge this PR in 6.4, adding a new method is acceptable, so no worries.